aten/src/ATen/native/cpu/ReduceOpsKernel.cpp

```diff
@@ -113,57 +114,6 @@ static void cumprod_cpu_kernel(const Tensor& result, const Tensor& self, int64_t
     );
   });
 }
-// custom min and max to be used in logcumsumexp for complex arguments
```

Comment:
Actually, why are we shuffling things around? Could you revert this change? Often things are structured certain way to improve compilation times (C++ is best compiled in large files, while CUDA is best compiled in smaller files).

Reply:
I'm moving this part of the code because it's going to be used in 2 cpp places: in ReduceOpsKernel.cpp and in BinaryOpsKernel.cpp. If I reverted this change, I'm not sure if I can load the function defined in ReduceOpsKernel.cpp to be used in BinaryOpsKernel.cpp.